packages/design-system/src/components/FieldContainer/FieldContainer.tsx

```diff
@@ -97,23 +113,38 @@ export class FieldContainer extends React.Component<FieldContainerProps> {
     } = this.props;
 
     const ComponentType = component;
-    const classes =
-      ComponentType === 'fieldset' ? classNames('ds-c-fieldset', className) : className;
+    const isFieldset = ComponentType === 'fieldset'
+    const classes = classNames({
+      'ds-c-fieldset': isFieldset
+    }, className);
+    const bottomError = errorPlacement === 'bottom' && errorMessage;
 
     // Field input props handled by <FieldContainer>
-    const fieldInputProps = {
+    const fieldInputProps = { 
       id: this.id,
       labelId: this.labelId,
+      errorId: this.errorId,
       setRef: this.setFieldRef,
     };
 
+    const renderBottomError = () => {
+      return (bottomError) ? (
+        <InlineError id={this.errorId} inversed={inversed}>
+          {errorMessage}
+        </InlineError>
+      ) : null;
+    }
+
     return (
-      <ComponentType className={classes}>
+      <ComponentType className={classes} >
         <FormLabel
           className={labelClassName}
           component={labelComponent}
-          errorMessage={errorMessage}
-          fieldId={this.id}
+          errorMessage={errorPlacement === 'top' ? errorMessage : null}
```

Comment:
Avoid using the `errorMessage` prop in `FormLabel` when using the bottom placement

Reply:
When the errorMessage is placed on the bottom we handle it inside `FieldContainer` [here](https://github.com/CMSgov/design-system/pull/914/files/7c88fbfe9f3ab2fdd10910cd6dac08d9734b5881#diff-68aab052f9429968f245cd29037c4ead7ebaf1192a95cea8aec887b86e5e970cR156). When its top placement, we pass errorMessage into `FormLabel` to be rendered